.github/workflows/update-bidi-types.yml

```diff
@@ -20,6 +26,7 @@ jobs:
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v3.5.3
         with:
           repository: w3c/webdriver-bidi
```

Comment:
we can also add a parameter for the repo to support forks

Reply:
good point. Done